### PR TITLE
Changelogs for Mender 3.7.0.

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: 10.Mender-server/docs.md [hunk 1]
```diff
@@ -7,6 +7,893 @@ shortcode-core:
...
+* Redis key prefix configuration not applied
+  ([MEN-6926](https://northerntech.atlassian.net/browse/MEN-6926))
+
+##### Features
+
+* Allow tenant token to be an opaque string
+  ([MEN-6502](https://northerntech.atlassian.net/browse/MEN-6502))
+* Added multiplaform build
+  ([QA-613](https://northerntech.atlassian.net/browse/QA-613))
+* Added multiplaform build
```
Comment (oldgiova) [inline 10.Mender-server/docs.md:348]: Duplicated lines

File: 10.Mender-server/docs.md [hunk 2]
```diff
@@ -7,6 +7,893 @@ shortcode-core:
...
+          Built-in permissions; keys are role names, values are
+          PermissionSetsWithScope from the API
+        additionalProperties:
+          $ref: '#/components/schemas/PermissionSetsWithScope'
+  ```
+
+  Ticket: ALV-67
+* Added multiplatform build
+  ([QA-613](https://northerntech.atlassian.net/browse/QA-613))
+* Added multiplatform build
```
Comment (oldgiova) [inline 10.Mender-server/docs.md:801]: Duplicated lines

File: 10.Mender-server/docs.md [hunk 3]
```diff
@@ -7,6 +7,893 @@ shortcode-core:
...
+  ([MEN-6857](https://northerntech.atlassian.net/browse/MEN-6857))
+* Missing path prefix from hubspot URLs
+
+##### Features
+
+* do not create the user in the create_organization workflow
+  ([MEN-5733](https://northerntech.atlassian.net/browse/MEN-5733))
+* Added multiplatform build
+  ([QA-613](https://northerntech.atlassian.net/browse/QA-613))
+* Added multiplatform build
```
Comment (oldgiova) [inline 10.Mender-server/docs.md:885]: Duplicated lines

**Review by oldgiova — APPROVED** [review]
Just some duplicated lines, rest is fine: pre-approving: :+1:

File: 52.mender-gateway/docs.md [hunk 4]
```diff
@@ -7,6 +7,54 @@ shortcode-core:
 github: false
 ---
 
+## mender-gateway 1.2.0
+
+_Released 12.28.2023_
+
+### Statistics
```
Comment (lluiscampos) [inline 52.mender-gateway/docs.md:14]: For the private repos we don't publish the statistics, only the changelog.

Comment (lluiscampos) [reply 52.mender-gateway/docs.md:14]: Follow-up here:
*  https://github.com/mendersoftware/mender-docs-changelog/pull/62

**Pull request merged**